js/lesson2/tutorial.md

```diff
@@ -105,8 +105,7 @@ var notEqual = apples !== oranges;
 console.log('Apples and oranges are different: ' + notEqual);
 ```
 
-> You may also see `==` and `!=`, these are similar but have some quirks so are
-> often not recommended.
+> You may also see `==` and `!=`, these are similar but have some quirks it's generally recommended to avoid them.
```

Comment:
> these are similar but have some quirks it's generally recommended to avoid them.

Missed the word 'so'. Should probably read 
> these are similar but have some quirks, **so** it's generally recommended to avoid them.